src/solvers/dgsem_tree/subcell_limiters.jl

```diff
@@ -199,8 +199,14 @@ function Base.show(io::IO, ::MIME"text/plain", limiter::SubcellLimiterIDP)
                 ]
             end
             if positivity
-                string = "Positivity limiting for conservative variables $(limiter.positivity_variables_cons) and $(limiter.positivity_variables_nonlinear)"
-                setup = [setup..., "" => string]
+                if !isempty(limiter.positivity_variables_cons)
+                    string = "conservative variables $(limiter.positivity_variables_cons)"
+                    setup = [setup..., "" => "Positivity limiting for " * string]
+                end
+                if !isempty(limiter.positivity_variables_nonlinear)
+                    string = "$(limiter.positivity_variables_nonlinear)"
+                    setup = [setup..., "" => "Positivity limiting for " * string]
+                end
```

Comment:
Since `setup` is a `Vector`, a more canonical way of extending it would be to use `push!`:
```suggestion
                if !isempty(limiter.positivity_variables_cons)
                    string = "conservative variables $(limiter.positivity_variables_cons)"
                    push!(setup, "" => "Positivity limiting for " * string)
                end
                if !isempty(limiter.positivity_variables_nonlinear)
                    string = "$(limiter.positivity_variables_nonlinear)"
                    push!(setup, "" => "Positivity limiting for " * string)
                end
```

Reply:
Thanks. Done in [9f185f2](https://github.com/trixi-framework/Trixi.jl/pull/1954/commits/9f185f2d93b363c39f6a6aec856b297518a0c373)